### PR TITLE
adding serverless and consuming lambdas locally

### PR DESCRIPTION
**Pull request by diegodafm**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+service: taco-gallery
```
Comment (levymoreira) [inline serverless.yml]: Rename the service name

File: serverless.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            integration: lambda
+            cors: true
+
+
+plugins:
+  # - serverless-dynamodb-local
+  - serverless-offline
+  # - serverless-webpack
+
+# custom:
```
Comment (levymoreira) [inline serverless.yml:61]: We can leave this commented, maybe be needed for the real project but not here

File: serverless.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          path: hello
+          method: post
+          cors: true
+  handleMessage:
+      handler: index.handleMessage
+      events:
+        - http:
+            path: handleMessage
+            method: post
+            integration: lambda
```
Comment (levymoreira) [inline serverless.yml:54]: add `integration: lambda` to sayHello too

File: src/lambdas/sayHello.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+const {isOffline} = require('../helpers/env');
+const {post} = require('../helpers/http');
+
+exports.sayHello = (event, context, callback) => {
+    const { name } = JSON.parse(event.body);
+    if (!name) {
+        callback('No name received');
+    }
+
+    if (isOffline) {
```
Comment (levymoreira) [inline src/lambdas/sayHello.js:11]: We can create a service called 'remoteLambda' with a function `invoke` and make this verification there.

File: src/lambdas/sayHello.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+const AWS = require('aws-sdk');
+const {isOffline} = require('../helpers/env');
+const {post} = require('../helpers/http');
+
+exports.sayHello = (event, context, callback) => {
+    const { name } = JSON.parse(event.body);
```
Comment (levymoreira) [inline src/lambdas/sayHello.js]: When add the integration:lambda I believe that we can remove the `JSON.parse`

File: src/lambdas/sayHello.js (hunk 4 above)
Comment (diegodafm) [reply src/lambdas/sayHello.js:11]: I'm gonna cover this part in another story

File: serverless.yml (hunk 3 above)
Comment (diegodafm) [reply serverless.yml:54]: done!

File: serverless.yml (hunk 2 above)
Comment (diegodafm) [reply serverless.yml:61]: we are using it soon to integrate with dynamoDB

**Pull request merged**